StRoot/StEvent/StTriggerData2022.cxx

```diff
@@ -1421,16 +1426,17 @@ unsigned short StTriggerData2022::fmsADC(int crt, int adr, int ch, int prepost)
 unsigned short StTriggerData2022::fmsTDC(int crt, int adr, int ch, int prepost) const
 {
     int buffer = prepostAddress(prepost);
-    if (buffer >= 0 && crt>=0 && crt<=7 && adr>=0 && adr<16 && ch>=0 && ch<=31){
+    if (buffer >= 0 && crt>=0 && crt<=8 && adr>=0 && adr<16 && ch>=0 && ch<=31){
         switch(crt){
             case 0: return tbbq[buffer][adr][ch];
-            case 1: return tqt1[buffer][adr][ch];
```

Comment:
Same comment (although this time case 0 can be reached).

Reply:
same as above